docs/tutorial_trajectories.md

```diff
@@ -1,73 +1,89 @@
 ## Setting up an OPTIMADE Trajectory database.
 
 The tutorial explains how to set up an OPTIMADE server for trajectory data.
-The code linked to here matches the trajectory endpoint as described in [https://github.com/JPBergsma/OPTIMADE/blob/Trajectory_proposal_v0.1/optimade.rst](https://github.com/JPBergsma/OPTIMADE/blob/Trajectory_proposal_v0.1/optimade.rst) and discussed in [OPTIMADE PR#377](https://github.com/Materials-Consortia/OPTIMADE/pull/377):
+The code linked to here matches the trajectory endpoint as described in [the trajectories proposal](https://github.com/JPBergsma/OPTIMADE/blob/Trajectory_proposal_v0.1/optimade.rst) and discussed in [OPTIMADE PR#377](https://github.com/Materials-Consortia/OPTIMADE/pull/377).
 This corresponds to [version 1.1 of the OPTIMADE specification](https://github.com/Materials-Consortia/OPTIMADE/releases/tag/v1.1.0).
-The trajectory endpoint is however still under development and there will probably be significant changes before it is merged with the main OPTIMADE specification.  
+The trajectory endpoint is however still under development and there may still be significant changes before it is merged with the main OPTIMADE specification, based on feedback from the community.
```

Comment:
"may still" is not strong enough. The chance that significant changes will be made is much larger than 50%. So I think I'll stick with "will probably" 

```suggestion
The trajectory endpoint is however still under development and there will probably be significant changes before it is merged with the main OPTIMADE specification, based on feedback from the community.
```